pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -1152,6 +1152,16 @@ public void writeBundleDataOnZooKeeper() {
         for (Map.Entry<String, BundleData> entry : loadData.getBundleData().entrySet()) {
             final String bundle = entry.getKey();
             final BundleData data = entry.getValue();
+            if (((conf.getLoadBalancerBundleThroughputThresholdInByte() > 0
+                    && data.getLongTermData().getMsgThroughputIn()
+                    < conf.getLoadBalancerBundleThroughputThresholdInByte())
+                    || (conf.getLoadBalancerBundleMsgThreshold() > 0
+                    && data.getLongTermData().getMsgRateIn() < conf.getLoadBalancerBundleMsgThreshold()))
+                    && bundlesCache.exists(getBundleDataPath(bundle)).join()) {
```

Comment:
maybe we can use `bundlesCache.getChildren` to get current exists bundle data on zk to reduce zk exist call